docs/walkthrough.rst

```diff
@@ -0,0 +1,199 @@
+===========
+Walkthrough
+===========
+
+The aim of the four Annotation scripts is to provide a way to edit \
+Key-Value pairs (MapAnnotations) and Tags (TagAnnotations) by batch:
+
+* Import Key-Value pairs and Tags as Annotations from a .csv file
+* Export Key-Value pairs and Tags to a .csv file
+* Delete Key-Value pairs
+* Convert the namespaces of Key-Value pairs
+
+Following this walkthrough, we hope to give you an understanding of \
+the four scripts that will help you to manipulate your annotations.
+
+Object selection with the scripts
+---------------------------------
+
+The object selection logic is the same for all four scripts. In OMERO there are \
+two distinct hierarchies:
+
+* **Projects** → **Datasets** → **Images**
+* **Screens** → **Plates** → (**Wells** or **Acquisition/Run**) → **Images**
+
+More details about selecting High-Content-Screening are given in another \
+section (see :doc:`Selecting High-Content-Screening </extended_description>`).
+
+To select which objects to manipulate by batch, the scripts offer four distinct ways.
+
+Direct selection
+^^^^^^^^^^^^^^^^
+The first is to simply select the all the desired objects. Opening the script \
+after selecting the object will prefill the script parameters with the right \
+object type and IDs:
+
+.. figure:: images/annotation_01_selecting_all_autofill.png
+
+   *Selection of multiple datasets and the auto-filled Export to CSV script.*
+..
+
+For the direct selection, leave the Target Data Type as "*<selected>*" \
+(in this example, choosing **Dataset** would also work).
+
+Children selection
+^^^^^^^^^^^^^^^^^^
+Instead of selecting the objects one by one, we can select the parent object \
+and set the "*Target Data Type*" to the type of the children objects \
+we want to select.
+
+.. figure:: images/annotation_02_selecting_parentchild_autofill.png
+
+   *Selection of all the images of a project and the auto-filled Export to CSV script.*
+..
+
+In this example, we would select all the **Images** found under the **Project:701**.
+
+Tag selection
+^^^^^^^^^^^^^
+The third option is to select from a tag all the objects of a given type \
+(do not confiuse with the option of exporting tags). This grants additional  \
+flexibility, either to select objects attached to different projects \
+or different owners, or to have a finer control over \
+which objects to process.
+
+.. figure:: images/annotation_03_selecting_tags.png
+
+   *Selection of a subset of datasets from a tag and the auto-filled Export to CSV script.*
+..
+
+The selection works in the same way as for the children selection. Note \
```

Comment:
Which is which way ? Be sure, I forgot it already from reading the children selection paragraph.

Reply:
changed to describe the example in the picture above